server/routes/createGoal/createGoalController.test.ts

```diff
@@ -58,6 +59,40 @@ describe('createGoalController', () => {
     errors = []
   })
 
+  describe('getCreateGoalView', () => {
```

Comment:
This test was missing, so I've added it (because I had to make a change to the implementation in the controller, so made sense to test it)